lib/PuppeteerSharp/BrowserOptions.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace PuppeteerSharp
```

Comment:
I'd feel better using an interface here, there is no logic we're reusing here, it looks more like a contract.